src/modules/simulator/simulator.cpp

```diff
@@ -163,12 +163,14 @@ int Simulator::start(int argc, char *argv[])
 			_instance->initializeSensorData();
 #ifndef __PX4_QURT
 			// Update sensor data
-			_instance->pollForMAVLinkMessages(false);
+			_instance->set_publish(false);
```

Comment:
@bkueng, case convention is just to match other methods in the class.

Reply:
Yup, good.